### PR TITLE
Run timers bug fix

### PR DESCRIPTION
**Pull request by jorgemiguelreis**

### CODE REVIEW

File: src/static/js/pages/run/activeRun.js [hunk 1]
```diff
@@ -37,12 +37,14 @@ var activeRun = function(element) {
         var moveButton = jQuery(".button-move", matchedObject);
         var continueButton = jQuery(".button-continue", matchedObject);
         var killButton = jQuery(".button-kill", matchedObject);
+        console.log("NEW");
```
Comment (marcosoliveira91) [inline src/static/js/pages/run/activeRun.js]: please, remove this

File: src/static/js/pages/run/activeRun.js [hunk 2]
```diff
@@ -37,12 +37,14 @@ var activeRun = function(element) {
...
         var killButton = jQuery(".button-kill", matchedObject);
+        console.log("NEW");
 
         /** Timers **/
         var sations = jQuery(".stations", element);
         var stationsNum = sations.attr("data-stations_num");
         var runTimerElement = jQuery(".run-timer", element);
-        var stationTimersElement = jQuery(".station-timer", element);
+        var stationTimersElement = jQuery(".stations-container", element);
+        console.log(stationTimersElement);
```
Comment (marcosoliveira91) [inline src/static/js/pages/run/activeRun.js]: and this

File: src/static/js/pages/run/activeRun.js [hunk 3]
```diff
@@ -59,23 +61,31 @@ var activeRun = function(element) {
         taktTimer.addEventListener('secondsUpdated', function () {
             taktTimerElement.html(taktTimer.getTimeValues().toString());
         });
+        console.log(stationTimersElement);
```
Comment (marcosoliveira91) [inline src/static/js/pages/run/activeRun.js]: same

File: src/static/js/pages/run/activeRun.js [hunk 4]
```diff
@@ -59,23 +61,31 @@ var activeRun = function(element) {
...
         for (var t = 0, length = stationTimers.length; t < length; t++) {
+            let timerIndex = t;
             var timerNum = t + 1;
-            stationTimers[t].addEventListener('secondsUpdated', function () {
+
+            stationTimers[timerIndex].addEventListener('secondsUpdated', function () {
                 var element = jQuery('.station-timer-' + timerNum, stationTimersElement);
-                element.html(stationTimers[t].getTimeValues().toString());
+                let currentTimer = stationTimers[timerIndex];
+                console.log(stationTimersElement);
```
Comment (marcosoliveira91) [inline src/static/js/pages/run/activeRun.js]: and these

File: src/static/js/pages/run/activeRun.js (hunk 3 above)
Comment (jorgemiguelreis) [reply src/static/js/pages/run/activeRun.js]: done

**Pull request merged**